src/Styling/SubMenu.css

```diff
@@ -0,0 +1,41 @@
+.Menu_item_links {
+  display: flex;
+  color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  list-style: none;
+  height: 60px;
+  text-decoration: none;
+  font-size: 18px;
+  font-family: 'Poppins';
+}
+
+.Menu_item_links:hover { 
+    background: #252831;
+    border-left: 4px solid #DE5B31 ;
+    cursor: pointer;
+}
+
+.Menu_items {
+    margin-left: 15px;
+}
+
+.DropdownLink {
```

Comment:
in all other classNames you’re using snakeCase. I would make this name snakeCase as well